### ISSUE TITLE
Prebid 4.+ and auction without GDPR consent when ​consentManagement module timeout exceeded

### ISSUE THREAD

**Issue by weuron** [prebid/Prebid.js#7984]
## Type of issue
Feature request.

## Description
Since Prebid 4.+ was introduced there is no posibility use the variable "allowAuctionWithoutConsent: true" in consentManagement module.

When a user in EU destination (defaultGdprScope: true) spends on web site more time than alowed timeout of consentManagement module the Prebid falls to error "CMP workflow exceeded timeout threshold. Canceling auction as per consentManagement config."

## Steps to reproduce
My setting of consentManagement in Prebid 4.43.4 is:
    consentManagement: {
        gdpr: {
          cmpApi: 'iab',
          timeout: 8000,
          defaultGdprScope: true
        }
    }

### Actual results
The Prebid falls to error "CMP workflow exceeded timeout threshold. Canceling auction as per consentManagement config."

### Expected results
Is it posible with the Prebid 4.+ or higher version and the consentManagement module to setup in the way that after timeout of consentManagement the Prebid will continue with auction without GDPR consent in the same way how it was enabled with the variable "allowAuctionWithoutConsent: true" on the all versions Prebid 3?

**Comment by dgirardi** [comment]
This looks like a duplicate of https://github.com/prebid/Prebid.js/issues/7156 (also contains an example of the configuration to use), which was addressed in https://github.com/prebid/Prebid.js/pull/7769 (merged in 6.5.0).

**Comment by gglas** [comment]
@weuron did @dgirardi 's comment resolve your issue?  let us know and we can close out.

**Comment by weuron** [comment]
Thanks, in the case [#7156](https://github.com/prebid/Prebid.js/issues/7156) the same error is well described but I don't understand why the case is closed? Because in our tests for Prebid 4.43.4 and also in the demo from communication [#7156](https://github.com/prebid/Prebid.js/issues/7156) (commented on 1 Dec 2021) with Prebid 5.2, see the online test here https://demo.cpex.cz/prebid/?pbjs_debug=true Prebid Error _"ERROR: CMP workflow exceeded timeout threshold. Canceling auction as per consentManagement config."_ is returned. See PRTSC [here](https://res.cloudinary.com/dryfbzbjb/image/upload/v1643620621/IM/cpex-2_aopznj.png).

`        pbjs.setConfig({
          consentManagement: {
            gdpr: {
              cmpApi: 'iab',
              defaultGdprScope: true,
              rules: [
                { purpose: 'storage', enforcePurpose: false, enforceVendor: false },
                { purpose: 'basicAds', enforcePurpose: false, enforceVendor: false }
              ]
            }
          },
        })`

    
Do you need more test data to compare the SSP behavior of Prebid 3 with the allowAuctionWithoutConsent parameter and the described errors for Prebid 4 and 5? Or do you think it's already fixed for Prebid 6?

**Comment by dgirardi** [comment]
@weuron from 6.5.0 the auction will not be cancelled after the CMP times out; if GDPR enforcement is also disabled (as is done in https://github.com/prebid/Prebid.js/issues/7156) that will allow it to continue and you should not see that error. The SSP then has the option of bidding, but many do not if they see GDPR is in scope but no consent data is available.

**Comment by weuron** [comment]
Perfect that sounds good. Please keep the case open till I finish the testing of version 6.5.0. I will let you know once it is done.

**Comment by weuron** [comment]
Perfect - with version 6.5.0. the auction will not be cancelled after the CMP times out.

After time out Prebid is returning only "_WARNING: No response from CMP, continuing auction..._" (PRTSC [here](https://res.cloudinary.com/dryfbzbjb/image/upload/v1643752912/IM/gdpr-P6-2_ntcdff.png))
 and continues without any errors.

With "Bids Requested" is in the variable "gdprConsent" sent "gdprApplies: true" (PRTSC [here](https://res.cloudinary.com/dryfbzbjb/image/upload/v1643753662/IM/gdpr-P6-3_ga1f29.png)).

Thaks so mutch, this is all what we needed. You can close this case.
